<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
  <testsuite errors="0" failures="2" hostname="LYUR" id="0" name="EnrollmentsControllerTests" package="gradadvising" tests="8" time="9.194" timestamp="2013-07-30T15:04:16">
      <properties />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testIndex" time="4.502" />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testList" time="2.412" />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testCreate" time="0.236" />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testSave" time="1.287">
          <failure message="Assertion failed: &#xa;&#xa;assert model.enrollmentsInstance != null&#xa;       |     |                   |&#xa;       [:]   null                false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert model.enrollmentsInstance != null
       |     |                   |
       [:]   null                false

	at gradadvising.EnrollmentsControllerTests.testSave(EnrollmentsControllerTests.groovy:40)
</failure>

      </testcase>

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testShow" time="0.192" />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testEdit" time="0.087" />

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testUpdate" time="0.13">
          <failure message="Assertion failed: &#xa;&#xa;assert view == &quot;/enrollments/edit&quot;&#xa;       |    |&#xa;       null false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert view == &quot;/enrollments/edit&quot;
       |    |
       null false

	at gradadvising.EnrollmentsControllerTests.testUpdate(EnrollmentsControllerTests.groovy:108)
</failure>

      </testcase>

      <testcase classname="gradadvising.EnrollmentsControllerTests" name="testDelete" time="0.196" />

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="1" name="EnrollmentsTests" package="gradadvising" tests="1" time="0.257" timestamp="2013-07-30T15:04:25">
      <properties />

      <testcase classname="gradadvising.EnrollmentsTests" name="testSomething" time="0.121">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.EnrollmentsTests.testSomething(EnrollmentsTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="2" hostname="LYUR" id="2" name="GradeControllerTests" package="gradadvising" tests="8" time="1.185" timestamp="2013-07-30T15:04:26">
      <properties />

      <testcase classname="gradadvising.GradeControllerTests" name="testIndex" time="0.396" />

      <testcase classname="gradadvising.GradeControllerTests" name="testList" time="0.068" />

      <testcase classname="gradadvising.GradeControllerTests" name="testCreate" time="0.064" />

      <testcase classname="gradadvising.GradeControllerTests" name="testSave" time="0.119">
          <failure message="Assertion failed: &#xa;&#xa;assert model.gradeInstance != null&#xa;       |     |             |&#xa;       [:]   null          false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert model.gradeInstance != null
       |     |             |
       [:]   null          false

	at gradadvising.GradeControllerTests.testSave(GradeControllerTests.groovy:40)
</failure>

      </testcase>

      <testcase classname="gradadvising.GradeControllerTests" name="testShow" time="0.074" />

      <testcase classname="gradadvising.GradeControllerTests" name="testEdit" time="0.081" />

      <testcase classname="gradadvising.GradeControllerTests" name="testUpdate" time="0.112">
          <failure message="Assertion failed: &#xa;&#xa;assert view == &quot;/grade/edit&quot;&#xa;       |    |&#xa;       null false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert view == &quot;/grade/edit&quot;
       |    |
       null false

	at gradadvising.GradeControllerTests.testUpdate(GradeControllerTests.groovy:108)
</failure>

      </testcase>

      <testcase classname="gradadvising.GradeControllerTests" name="testDelete" time="0.117" />

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="3" name="GradeTests" package="gradadvising" tests="1" time="0.217" timestamp="2013-07-30T15:04:27">
      <properties />

      <testcase classname="gradadvising.GradeTests" name="testSomething" time="0.081">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.GradeTests.testSomething(GradeTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="LYUR" id="4" name="ProgramControllerTests" package="gradadvising" tests="8" time="1.277" timestamp="2013-07-30T15:04:27">
      <properties />

      <testcase classname="gradadvising.ProgramControllerTests" name="testIndex" time="0.484" />

      <testcase classname="gradadvising.ProgramControllerTests" name="testList" time="0.057" />

      <testcase classname="gradadvising.ProgramControllerTests" name="testCreate" time="0.077" />

      <testcase classname="gradadvising.ProgramControllerTests" name="testSave" time="0.116">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/program/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@fe55e8&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/program/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@fe55e8

	at gradadvising.ProgramControllerTests.testSave(ProgramControllerTests.groovy:48)
</failure>

      </testcase>

      <testcase classname="gradadvising.ProgramControllerTests" name="testShow" time="0.114">
          <failure message="Assertion failed: &#xa;&#xa;assert program.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert program.save() != null
       |       |      |
       null    null   false

	at gradadvising.ProgramControllerTests.testShow(ProgramControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="gradadvising.ProgramControllerTests" name="testEdit" time="0.1">
          <failure message="Assertion failed: &#xa;&#xa;assert program.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert program.save() != null
       |       |      |
       null    null   false

	at gradadvising.ProgramControllerTests.testEdit(ProgramControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="gradadvising.ProgramControllerTests" name="testUpdate" time="0.134">
          <failure message="Assertion failed: &#xa;&#xa;assert program.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert program.save() != null
       |       |      |
       null    null   false

	at gradadvising.ProgramControllerTests.testUpdate(ProgramControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="gradadvising.ProgramControllerTests" name="testDelete" time="0.08">
          <failure message="Assertion failed: &#xa;&#xa;assert program.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert program.save() != null
       |       |      |
       null    null   false

	at gradadvising.ProgramControllerTests.testDelete(ProgramControllerTests.groovy:144)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="5" name="ProgramTests" package="gradadvising" tests="1" time="0.257" timestamp="2013-07-30T15:04:28">
      <properties />

      <testcase classname="gradadvising.ProgramTests" name="testSomething" time="0.133">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.ProgramTests.testSomething(ProgramTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="LYUR" id="6" name="StudentControllerTests" package="gradadvising" tests="8" time="1.113" timestamp="2013-07-30T15:04:29">
      <properties />

      <testcase classname="gradadvising.StudentControllerTests" name="testIndex" time="0.36" />

      <testcase classname="gradadvising.StudentControllerTests" name="testList" time="0.083" />

      <testcase classname="gradadvising.StudentControllerTests" name="testCreate" time="0.066" />

      <testcase classname="gradadvising.StudentControllerTests" name="testSave" time="0.085">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/student/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@14fe17&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/student/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@14fe17

	at gradadvising.StudentControllerTests.testSave(StudentControllerTests.groovy:48)
</failure>

      </testcase>

      <testcase classname="gradadvising.StudentControllerTests" name="testShow" time="0.113">
          <failure message="Assertion failed: &#xa;&#xa;assert student.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       null null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert student.save() != null
       |       |      |
       |       null   false
       null null

	at gradadvising.StudentControllerTests.testShow(StudentControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="gradadvising.StudentControllerTests" name="testEdit" time="0.102">
          <failure message="Assertion failed: &#xa;&#xa;assert student.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       null null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert student.save() != null
       |       |      |
       |       null   false
       null null

	at gradadvising.StudentControllerTests.testEdit(StudentControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="gradadvising.StudentControllerTests" name="testUpdate" time="0.089">
          <failure message="Assertion failed: &#xa;&#xa;assert student.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       null null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert student.save() != null
       |       |      |
       |       null   false
       null null

	at gradadvising.StudentControllerTests.testUpdate(StudentControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="gradadvising.StudentControllerTests" name="testDelete" time="0.084">
          <failure message="Assertion failed: &#xa;&#xa;assert student.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       null null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert student.save() != null
       |       |      |
       |       null   false
       null null

	at gradadvising.StudentControllerTests.testDelete(StudentControllerTests.groovy:144)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="7" name="StudentTests" package="gradadvising" tests="1" time="0.25" timestamp="2013-07-30T15:04:30">
      <properties />

      <testcase classname="gradadvising.StudentTests" name="testSomething" time="0.12">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.StudentTests.testSomething(StudentTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="LYUR" id="8" name="SubjectControllerTests" package="gradadvising" tests="8" time="1.148" timestamp="2013-07-30T15:04:30">
      <properties />

      <testcase classname="gradadvising.SubjectControllerTests" name="testIndex" time="0.363" />

      <testcase classname="gradadvising.SubjectControllerTests" name="testList" time="0.067" />

      <testcase classname="gradadvising.SubjectControllerTests" name="testCreate" time="0.058" />

      <testcase classname="gradadvising.SubjectControllerTests" name="testSave" time="0.093">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/subject/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@1f43093&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/subject/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@1f43093

	at gradadvising.SubjectControllerTests.testSave(SubjectControllerTests.groovy:48)
</failure>

      </testcase>

      <testcase classname="gradadvising.SubjectControllerTests" name="testShow" time="0.092">
          <failure message="Assertion failed: &#xa;&#xa;assert subject.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert subject.save() != null
       |       |      |
       null    null   false

	at gradadvising.SubjectControllerTests.testShow(SubjectControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="gradadvising.SubjectControllerTests" name="testEdit" time="0.089">
          <failure message="Assertion failed: &#xa;&#xa;assert subject.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert subject.save() != null
       |       |      |
       null    null   false

	at gradadvising.SubjectControllerTests.testEdit(SubjectControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="gradadvising.SubjectControllerTests" name="testUpdate" time="0.103">
          <failure message="Assertion failed: &#xa;&#xa;assert subject.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert subject.save() != null
       |       |      |
       null    null   false

	at gradadvising.SubjectControllerTests.testUpdate(SubjectControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="gradadvising.SubjectControllerTests" name="testDelete" time="0.154">
          <failure message="Assertion failed: &#xa;&#xa;assert subject.save() != null&#xa;       |       |      |&#xa;       null    null   false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert subject.save() != null
       |       |      |
       null    null   false

	at gradadvising.SubjectControllerTests.testDelete(SubjectControllerTests.groovy:144)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="9" name="SubjectTests" package="gradadvising" tests="1" time="0.256" timestamp="2013-07-30T15:04:31">
      <properties />

      <testcase classname="gradadvising.SubjectTests" name="testSomething" time="0.122">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.SubjectTests.testSomething(SubjectTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="2" hostname="LYUR" id="10" name="YearSemControllerTests" package="gradadvising" tests="8" time="0.976" timestamp="2013-07-30T15:04:32">
      <properties />

      <testcase classname="gradadvising.YearSemControllerTests" name="testIndex" time="0.292" />

      <testcase classname="gradadvising.YearSemControllerTests" name="testList" time="0.085" />

      <testcase classname="gradadvising.YearSemControllerTests" name="testCreate" time="0.068" />

      <testcase classname="gradadvising.YearSemControllerTests" name="testSave" time="0.078">
          <failure message="Assertion failed: &#xa;&#xa;assert model.yearSemInstance != null&#xa;       |     |               |&#xa;       [:]   null            false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert model.yearSemInstance != null
       |     |               |
       [:]   null            false

	at gradadvising.YearSemControllerTests.testSave(YearSemControllerTests.groovy:40)
</failure>

      </testcase>

      <testcase classname="gradadvising.YearSemControllerTests" name="testShow" time="0.076" />

      <testcase classname="gradadvising.YearSemControllerTests" name="testEdit" time="0.058" />

      <testcase classname="gradadvising.YearSemControllerTests" name="testUpdate" time="0.091">
          <failure message="Assertion failed: &#xa;&#xa;assert view == &quot;/yearSem/edit&quot;&#xa;       |    |&#xa;       null false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert view == &quot;/yearSem/edit&quot;
       |    |
       null false

	at gradadvising.YearSemControllerTests.testUpdate(YearSemControllerTests.groovy:108)
</failure>

      </testcase>

      <testcase classname="gradadvising.YearSemControllerTests" name="testDelete" time="0.072" />

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="LYUR" id="11" name="YearSemTests" package="gradadvising" tests="1" time="0.066" timestamp="2013-07-30T15:04:33">
      <properties />

      <testcase classname="gradadvising.YearSemTests" name="testSomething" time="0.063">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at gradadvising.YearSemTests.testSomething(YearSemTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
</testsuites>
